llvm/include/llvm/ADT/bit.h

```diff
@@ -29,7 +29,7 @@
 
 #if defined(__linux__) || defined(__GNU__) || defined(__HAIKU__) ||            \
     defined(__Fuchsia__) || defined(__EMSCRIPTEN__) || defined(__NetBSD__) ||  \
-    defined(__OpenBSD__) || defined(__DragonFly__)
+    defined(__OpenBSD__) || defined(__DragonFly__) || defined(__wasm__)
```

Comment:
This should probably be `__wasi__` rather than `__wasm__` since the later is the architecture, which by itself doesn't mean we have `endian.h` available.